src/components/FbLoginButton/FbLoginButton.js

```diff
@@ -43,6 +43,14 @@ class FbLoginButton extends React.Component {
         global.alert('there was an error, please contact me')
       })
   }
+
+  responseApi = () => {
+    global.FB.api('/me?fields=id,name,picture',
```

Comment:
I don't think you need to call FB.API separately, just include those details in the scope i nthe login call